developer-docs-site/docs/reference/telemetry.md

```diff
@@ -3,32 +3,101 @@ title: "Telemetry"
 slug: "telemetry"
 ---
 
-At Aptos Labs, we develop software and services for the greater Aptos community and ecosystem. On top of community feedback, we use telemetry to help improve the decentralization of the network by understanding how our software is being deployed and run.
+When you operate a node on an Aptos network, your node can be set to send telemetry data to Aptos Labs. You can disable telemetry at any point. If telemetry remains enabled, Aptos node binary will send telemetry data in the background.
 
-The Aptos node binary collects telemetry such as software version, operating system information, and IP address. See [Types of information collected](#types-of-information-collected).
+The Aptos node binary running on your node collects telemetry data such as software version, operating system information and the IP address of your node. This telemetry data is used to enhance the decentralization of the network.
 
+:::tip No personal information is collected
 The Aptos node binary does **not** collect personal information such as usernames or email addresses.
+:::
 
-Users can disable telemetry at any point. If telemetry remains enabled, Aptos node binary will send telemetry data in the background.
+## Metrics collected
 
-# Disabling telemetry
+### Core metrics
 
-On macOS and Linux, you can disable telemetry by setting the `APTOS_DISABLE_TELEMETRY` environment variable to any value.
+- **Metrics**: All the [Prometheus](https://prometheus.io/) metrics that are collected within the node.
+- **Logs**: Logs of warn-level and higher level, with the ability to collect up to debug logs.
```

Comment:
Metrics could fall under multiple categories below including node information, network, system and build information metrics. I would just keep them separate from node metrics. Logs should be separate too in my opinion.